### DISCUSSION TITLE
How to set CCD2 per core curve optimiser ?

### DISCUSSION THREAD

**Discussion by jzhehe** [JamesCJ60/Universal-x86-Tuning-Utility#188] [General]
How to set CCD2 per core curve optimiser ?

**Comment by Edensystem** [comment]
You cannot with the current version of UXTU
A workaround is to use the global curve to affect CCD1 and CCD2 as a base start, and then adjust CCD1 with the « per core » to go further .
This is what I do on my 7945HX , and I get good result.
